RemoteSystemsTempFiles/.project

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>RemoteSystemsTempFiles</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.rse.ui.remoteSystemsTempNature</nature>
+	</natures>
```

Comment:
I have no idea, what this added Eclipse nature means for others, please get rid of that or we have to be sure that this does not  create issues for other Eclipse users. Maybe someone else has an idea what this is?